spec/controllers/session_controller_spec.rb

```diff
@@ -659,6 +659,22 @@ def post_login
         end
       end
     end
+
+    context 'rate limited' do
+      it 'rate limits login' do
+        SiteSetting.max_logins_per_ip_per_hour = 2
+        RateLimiter.stubs(:disabled?).returns(false)
```

Comment:
I think it is better that we avoid stubbing here. Maybe we can add a method into `RateLimiter` that will allow us to enable the rate limit when we want to?